backend/tournesol/tests/test_api_statistics.py

```diff
@@ -0,0 +1,133 @@
+
```

Comment:
We can remove the extra blank line here.

```suggestion
```

Reply:
Ok I made the change.